src/sentry/static/sentry/app/views/onboarding/projectSetup/firstEventIndicator.jsx

```diff
@@ -0,0 +1,140 @@
+import {keyframes} from 'emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import EventWaiter from 'app/views/onboarding/projectSetup/eventWaiter';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+const FirstEventIndicator = props => {
+  const {orgId, projectId} = props;
+
+  return (
+    <EventWaiter orgId={orgId} projectId={projectId}>
+      {({firstIssue}) => (
+        <PoseGroup preEnterPose="init">
+          {!firstIssue ? (
+            <Waiting key="waiting" {...props} />
```

Comment:
Are `key`s needed here?

Reply:
Yeah react pose needs it:

> Every child must be provided a unique key property for PoseGroup to track entering and exiting children.

Even when not in an array.